core/block.go

```diff
@@ -29,6 +29,13 @@ func NewBlock(s Spec) *Block {
 		})
 	}
 
+	var sourceType SourceType
+	if s.Source != 0 {
+		sourceType = s.Source
+	} else {
+		sourceType = NONE
```

Comment:
I think probably this block just needs to be

```sourceType = s.Source```

then? Or, even, just put s.Source in the literal below?

On Mon, Jan 19, 2015 at 10:42 AM, nik hanselmann <notifications@github.com>
wrote:

> In core/block.go
> <https://github.com/nytlabs/st-core/pull/35#discussion-diff-23166906>:
>
> > @@ -29,6 +29,13 @@ func NewBlock(s Spec) *Block {
> >  		})
> >  	}
> >
> > +	var sourceType SourceType
> > +	if s.Source != 0 {
> > +		sourceType = s.Source
> > +	} else {
> > +		sourceType = NONE
>
> I believe NONE is 0, but semantically we are thinking NONE means "no
> source"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nytlabs/st-core/pull/35/files#r23166906>.
>